ecllibrary/std/DataPatterns/Profile.ecl

```diff
@@ -302,6 +326,19 @@ EXPORT Profile(inFile,
                     #APPEND(childDSFields, ',')
                 #END
                 #APPEND(childDSFields, %'fieldCount'% + ':' + %'fullName'%)
+                // Extract the child dataset into its own attribute so we can more easily
```

Comment:
Diff: New code as workaround, allowing a child dataset within a child record to be profiled.